src/Rings/mpoly-graded.jl

```diff
@@ -8,6 +8,7 @@
   multi_hilbert_series_parent::Generic.LaurentMPolyWrapRing{ZZRingElem, ZZMPolyRing}
 
   function MPolyDecRing(R::S, d::Vector{FinGenAbGroupElem}) where {S}
+    @req !(R isa MPolyDecRing) "cannot graded polynomial ring which is already decorated"
```

Comment:
```suggestion
    @req !(R isa MPolyDecRing) "cannot grade polynomial ring which is already decorated"
```
(Also needs to be adjusted in the doc example where the error is triggered.)